Assets/MRTK/Examples/Demos/Audio/Scenes/AudioLoFiEffectExamples.unity

```diff
@@ -498,8 +498,9 @@ GameObject:
   serializedVersion: 6
   m_Component:
   - component: {fileID: 336154624}
+  - component: {fileID: 336154625}
   m_Layer: 0
-  m_Name: SceneContent
+  m_Name: MixedRealtiySceneContent
```

Comment:
slight typo in the GO names: MixedReal*ti*ySceneContent

Reply:
T_T

Reply:
fixed